src/ethereum/info/polygonInfo.ts

```diff
@@ -216,7 +216,7 @@ const networkInfo: EthereumNetworkInfo = {
 
 const currencyInfo: EdgeCurrencyInfo = {
   canReplaceByFee: true,
-  currencyCode: 'MATIC',
+  currencyCode: 'POL',
```

Comment:
We should force a resync so the user doesn't have to. Search for "zksyncForceResyncUSDC" as an example

Reply:
Do we really need this? The `lastTransactionQueryHeight.POL` will simply be 0, because we have never seen POL before, and the user will automatically get the correct transactions. This would only be an issue if the user adds a custom token on Polygon called "MATIC", in which case they might see the wrong history.

Reply:
Yup you're right not necessary, my mistake